FAQs/Copy-fonts-to-linux-containers/README.md

```diff
@@ -0,0 +1,17 @@
+Copy necessary fonts to Linux containers
+----------------------------------------
+
+In Word to PDF conversion, Essential DocIO uses the fonts which are installed in the corresponding production machine to measure and draw the text. If the font is not available in the production environment, then the alternate font will be used to measure and draw text based on the environment. And so, it is mandatory to install all the fonts used in the Word document in machine to achieve proper preservation.
```

Comment:
Remove this para, we will convey only necessary things alone

Reply:
Addressed.